slider/mod.py

```diff
@@ -147,9 +147,29 @@ def circle_radius(cs):
     return (512 / 16) * (1 - 0.7 * (cs - 5) / 5)
 
 
+class HitWindows(namedtuple('HitWindows', 'perfect, good, bad')):
+    """Times to hit an object at various accuracies
+
+    Parameters
+    ----------
+    perfect : int
+        The maxumium number of milliseconds away from exactly on time a hit
+        can be to still be a 300
+    good : int
+        The maxumium number of milliseconds away from exactly on time a hit
+        can be to still be a 100
+    bad : int
+        The maxumium number of milliseconds away from exactly on time a hit
+        can be to still be a 50
+
+    Notes
+    -----
+    A hit further than the ``bad`` value away from exactly on time is a miss.
+    """
+
 def od_to_ms(od):
```

Comment:
could we add back an `od_to_ms_300` function because that special case is used already in slider. this would be a slight performance optimization for when you only need 300 and not all the different hit kinds.